lib/fs/fakefs_test.go

```diff
@@ -790,6 +793,9 @@ func testFakeFSCreateInsens(t *testing.T, fs Filesystem) {
 	if fd1.Name() != "fOo" {
 		t.Errorf("name of the file created as \"FOO\" is %s", fd1.Name())
 	}
+
+	fd1.Close()
```

Comment:
Should just defer acter open?

Reply:
Yep. And after every Create, apparently. Even after `_, err := Create`. I'm starting to hate windows.